zipline/gens/tradesimulation.py

```diff
@@ -256,13 +299,13 @@ def _process_snapshot(self, dt, snapshot, instant_fill):
             # Now that handle_data has been called and orders have been placed,
             # process the event stream to fill user orders based on the events
             # from this snapshot.
-            for event in events_to_be_processed:
-                for txn, order in blotter_process_trade(event):
+            for trade in events_to_be_processed:
+                for txn, order in blotter_process_trade(trade):
```

Comment:
No worries, a question could be 'do we need to keep this, considering the confusion that it causes in implementation'.
Not sure how many people are using the feature, we do not use it internally, as far as I know.